### PR TITLE
Please review this asap @casey-collab

### PR DESCRIPTION
**Pull request by AmirEskandari87**
Trump story enhancement

### CODE REVIEW

**Comment by AmirEskandari87** [comment]
Thanks @casey-collab

File: english/vote-the-president/trump/trump.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Are you ready for [WWIII](./WWIII/WWIII.md)?
 
 Know about [Latest hit from Trump Floyd "The Wall"](./trump-floyd-the-wall/thewall.md) !
 
+You must have a tiny.. pair of [hands](https://www.reddit.com/r/gifs/comments/48xv6z/tiny_hands_trump_oc/).
```
Comment (casey-collab) [inline english/vote-the-president/trump/trump.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/vote-the-president/trump/trump.md (hunk 1 above)
Comment (AmirEskandari87) [inline english/vote-the-president/trump/trump.md]: Link is working. @casey-collab

**Pull request closed without merging**